### ISSUE TITLE
Howto

### ISSUE THREAD

**Issue by zaggynl** [HolyAntimony/Reverstrix#1]
Could you create a short howto on getting this to work?
Had a go at it and got as far joining a room.

**Comment by zaggynl** [comment]
Found out so far:

Installation/prerequisies:
sudo pip3 install pygame pygame-menu matrix_client
python3 reversi.py (startup takes a while)

After "room <name !id"> appears use right/left arrows keys to switch room
Press enter to start in selected room.
To play use arrow keys to move green selection box, press enter to make move.

Player need to have power 50 otherwise cannot move?
`matrix_client.errors.MatrixRequestError: 403: {"errcode":"M_FORBIDDEN","error":"You don't have permission to post that to the room. user_level (0) < send_level (50)"}`

Eats a lot of CPU.

**Comment by Allymonies** [comment]
Yeah that's all correct, I'll add it to the readme, thanks! As for power level 50, you need to be able to send state events, which requires the permission labelled "Change Settings" (It'd probably be possible to send normal events, but then you run the risk of a client connecting and being unable to find the previous game event because it's buried under chat history.)

CPU Usage was a bug that should be fixed now.
